WebService/Startup.cs

```diff
@@ -71,7 +72,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             services.AddCors();
 
             // Add controllers as services so they'll be resolved.
-            services.AddMvc().AddControllersAsServices();
+            services.AddMvc().AddControllersAsServices().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
```

Comment:
do we need this additional function call? what's the downside of not having it? what's the benefit for having it?

Reply:
I'm adding it as per documentation here to get the latest behavior: https://docs.microsoft.com/en-us/aspnet/core/mvc/compatibility-version?view=aspnetcore-2.1

Else the default will set compatibility version to 2.0